packages/bierzo-wallet/src/routes/welcome/index.tsx

```diff
@@ -1,30 +1,71 @@
-import { Theme } from '@material-ui/core';
+import { Avatar, Button, Theme } from '@material-ui/core';
 import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import Image from 'medulas-react-components/lib/components/Image';
+import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
+import avatar from '../../assets/iov-logo.svg';
 import { history } from '../../store/reducers';
 import { PAYMENT_ROUTE } from '../paths';
 
 const useStyles = makeStyles((theme: Theme) => ({
   welcome: {
-    backgroundColor: theme.palette.primary.main,
+    backgroundColor: theme.palette.background.default,
     width: '100vw',
-    height: '100vh',
+    height: 'auto',
+    minHeight: '100vh',
     display: 'flex',
+    flexDirection: 'column',
     placeItems: 'center',
     placeContent: 'center',
   },
+
+  avatarRoot: {
+    backgroundColor: theme.palette.primary.main,
+    width: '200px',
+    height: '200px',
+  },
+
+  avatarIcon: {
+    width: '50%',
+  },
+
+  heading: {
+    margin: '40px 0 40px 0',
```

Comment:
Wrap `Typography` with a `Block` and use the `marginTop` and `marginBottom` prop, then, remove the margin css style here.

The less css we write on top level components, more descriptive and more semantic we do the views. Then a developer just reading the render method will understand what is going on. CSS details should be on low level components.